src/utils/emasser/outputFormatter.ts

```diff
@@ -80,9 +81,9 @@ function convertEpochToDateTime(dataObject: object): object {
  */
 export function outputFormat(data: object, doConversion = true): string {
   const conf = new ApiConfig()
-  const hideNulls: boolean = conf.displayNulls !== 'true'
-  const showEpoch: boolean = (conf.displayDateTime === 'true')
-  const debugging: boolean = (conf.debugging === 'true')
+  const hideNulls = !conf.displayNulls
```

Comment:
Does this still work because it seems like the previous check was checking against a string rather than boolean type?

Reply:
yes, this is correct, the configuration where these are set changed.